README.md

```diff
@@ -4,14 +4,49 @@
 [![Build Status](https://github.com/software-gardening/almanack/actions/workflows/pytest-tests.yml/badge.svg?branch=main)](https://github.com/software-gardening/almanack/actions/workflows/pytest-tests.yml?query=branch%3Amain)
 ![Coverage Status](https://raw.githubusercontent.com/software-gardening/almanack/main/media/coverage-badge.svg)
 
-An open-source handbook of applied guidance and tools for sustainable software development and maintenance.
+The Software Gardening Almanack is an open-source handbook of applied guidance and tools for sustainable software development and maintenance.
+The Almanack is available both as a book and a Python package.
 
-Please see the following links to access published content:
+## Handbook
 
 - __Online (HTML)__: https://software-gardening.github.io/almanack/
 - __Offline (PDF)__: [software-gardening-almanack.pdf](https://software-gardening.github.io/almanack/software-gardening-almanack.pdf)
 
-## CONTRIBUTING
+## Package
+
+You can install the Almanack with the following:
+
+```bash
+# install from pypi
+pip install almanack
+
+# install directly from source
+pip install git+https://github.com/software-gardening/almanack.git
+```
+
+Once installed, the Almanack may be used to analyze repositories for sustainable development practices.
```

Comment:
This might sound better?
```suggestion
Once installed, the Almanack can be used to analyze repositories for sustainable development practices.
```

Reply:
Thanks - yes, this sounds better as it's more of a utility than an authority from this standpoint.